plugins/infobright-bulk-loader/ui/src/main/java/org/pentaho/di/ui/trans/infobrightoutput/InfobrightLoaderDialog.java

```diff
@@ -66,7 +66,7 @@
  * Dialog box for the Infobright loader.
  */
 @PluginDialog( id = "InfobrightOutput", image = "BLKIB.svg", pluginType = PluginDialog.PluginType.STEP,
-        documentationUrl = "http://wiki.pentaho.com/display/EAI/InfoBright+Bulk+Loader" )
+        documentationUrl = "https://pentaho-community.atlassian.net/wiki/spaces/EAI/pages/386800365/Infobright+Loader" )
```

Comment:
see previous comment on documentationUrl

Reply:
Updated as per the message